CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - added an inverted look camera option (#700)
 - added a camera speed option for the manual camera (#815)
 - added an option to fix original texture issues (#826)
+- added menu specific controls (#814)
```

Comment:
Let's be more visible about this and mention that this brings arrows and enter support back. Ideally mention the version that took this away.